ghost/core/test/e2e-api/admin/explore.test.js

```diff
@@ -17,7 +17,8 @@ describe('Explore API', function () {
                 .expectStatus(200)
                 .matchBodySnapshot({
                     explore: {
-                        most_recently_published_at: anyISODateTime,
```

Comment:
@daniellockyer I actually do want to return a datetime, but somehow only got a date back from the db query. I couldn't make it work. Is there a better way of doing so?

Reply:
I think you want to wrap the output in a `Date` object, so if you update the signature of the `getMostRecentlyPublishedPostDate(): string` to `getMostRecentlyPublishedPostDate(): Date` everything should work as expected